tools/dev/loki-boltdb-storage-s3/config/loki.yaml

```diff
@@ -102,6 +102,7 @@ query_range:
         cache:
             memcached_client:
                 addresses: memcached:11211
+    accepted_query_response_format: protobuf
```

Comment:
Do not forget to set this configuration to enable Protobuf encoding between the querier and query frontend.